ctapipe/reco/hillas_intersection.py

```diff
@@ -43,6 +43,29 @@
     prefix="HillasIntersection",
 )
 
+FOV_ANGULAR_DISTANCE_LIMIT = 45 * u.deg
+
+
+def far_outside_fov(fov_lat, fov_lon):
```

Comment:
I'd make this private (i.e. add an underscore in front)

Reply:
Yes, no problem